pkg/epp/datalayer/metrics/spec_test.go

```diff
@@ -30,6 +30,10 @@ import (
 
 // --- Test Helpers ---
 
+func float64Ptr(v float64) *float64 {
+	return &v
+}
+
```

Comment:
no need for this function.
please reuse `"k8s.io/utils/ptr"`.
example can be seen in `conflgloader_test.go` file.